packages/core/src/components/label/label.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component, h, Prop } from '@stencil/core';
+import clsx from 'clsx';
+
+@Component({
+  tag: 'dso-label',
+  styleUrl: 'label.scss',
+  shadow: true
+})
+export class Label {
+  @Prop()
+  status?: 'primary' | 'info' | 'success' | 'warning' | 'danger';
+
+  private static statusMap = new Map<string, string>([
+    ['primary', 'Primair'],
+    ['info', 'Info'],
+    ['success', 'Succes'],
+    ['warning', 'Waarschuwing'],
+    ['danger', 'Gevaar']
+  ]);
+
+  render() {
+    return (
+      <span class={clsx('dso-label', { [`dso-label-${this.status}`]: this.status } )}>
+        <span class="sr-only">{this.status ? Label.statusMap.get(this.status) : `Label`}: </span>
```

Comment:
* `status` is optioneel. Als `!status` dan wordt er een `<span class="sr-only">Label</span>` getekend, terwijl het 

  Nunjucks component helemaal niets deed:
  ```html
  {% if status -%}<span class="sr-only">{{ status }}: </span>{% endif -%}
  ```

  De gegenereerde markup van het Nunjucks component en Web Component komt met dezelfde context niet overeen.
* Met `status = 'konijn';` wordt er een lege `<span class="sr-only"></span>` getekend, omdat `konijn` niet in de statusMap bestaat. In de render() moet eerst de waarde worden opgehaald en daarna voorwaardelijk worden getoond.